core/src/test/scala/io/qbeast/core/transform/LinearTransformationTest.scala

```diff
@@ -27,36 +27,43 @@ class LinearTransformationTest extends AnyFlatSpec with Matchers {
     an[IllegalArgumentException] should be thrownBy LinearTransformation(
       10000,
       0,
+      5000,
       IntegerDataType)
   }
 
   it should "save min and max values" in {
-    val linearT = LinearTransformation(0, 10000, IntegerDataType)
+    val linearT = LinearTransformation(0, 10000, 5000, IntegerDataType)
     linearT.minNumber should be(0)
     linearT.maxNumber should be(10000)
   }
 
+  // TODO check merge
   it should "merge transformations correctly" in {
-    val linearT = LinearTransformation(0, 10000, IntegerDataType)
+    val nullValue = 5000
+    val linearT = LinearTransformation(0, 10000, nullValue, IntegerDataType)
 
-    linearT.merge(LinearTransformation(0, 90000, IntegerDataType)) shouldBe LinearTransformation(
+    linearT.merge(
+      LinearTransformation(0, 90000, nullValue, IntegerDataType)) shouldBe LinearTransformation(
       0,
       90000,
+      nullValue,
       IntegerDataType)
 
     linearT.merge(
-      LinearTransformation(-100, 10000, IntegerDataType)) shouldBe LinearTransformation(
-      -100,
-      10000,
-      IntegerDataType)
+      LinearTransformation(
+        -100,
+        10000,
+        nullValue,
+        IntegerDataType)) shouldBe LinearTransformation(-100, 10000, nullValue, IntegerDataType)
 
     linearT.merge(
-      LinearTransformation(-100, 90000, IntegerDataType)) shouldBe LinearTransformation(
-      -100,
-      90000,
-      IntegerDataType)
+      LinearTransformation(
+        -100,
+        90000,
+        nullValue,
+        IntegerDataType)) shouldBe LinearTransformation(-100, 90000, nullValue, IntegerDataType)
 
-    linearT.merge(LinearTransformation(6, 9, IntegerDataType)) shouldBe linearT
+    linearT.merge(LinearTransformation(6, 9, 7, IntegerDataType)) shouldBe linearT
```

Comment:
Probably check the values instead of the whole Transformation?

Reply:
Yes, thank you!

Reply:
Solved on e8cdf06